packages/dates/src/utils.ts

```diff
@@ -116,6 +116,12 @@ export function validateISOWeek(week: ISOWeek): boolean {
   return date.isValid && date.year >= 2024 && date <= now;
 }
 
+export function getDraftSeasonEndDate(date: Date) {
```

Comment:
this method assumes draft season is just one week.. it would be better i think if it took a 'season' instead